apps/api_web/lib/api_web/views/vehicle_view.ex

```diff
@@ -84,4 +85,21 @@ defmodule ApiWeb.VehicleView do
   defp backwards_compatible_attributes(attributes, _, _) do
     attributes
   end
+
+  defp encode_carriages(%{carriages: nil} = vehicle) do
+    vehicle
```

Comment:
question: should this be an empty array rather than `nil`?

Reply:
My initial implementation actually was `[]`, but I changed to nil to keep consistent with the current behavior. It looks like the `nil` gets coerced to `[]` anyway, so I've gone back to that.